Decent.Minecraft.Client/Blocks/Coal.cs

```diff
@@ -11,6 +14,9 @@ protected Coal(bool charred) : base(BlockType.Coal)
         protected bool IsCharred { get; }
     }
 
+    /// <summary>
+    /// <a href="http://minecraft.gamepedia.com/Coal_Ore">Gamepedia link</a>.
+    /// </summary>
     public class Charcoal : Coal
```

Comment:
Leave `Coal` and `Charcoal` as they currently are, except make the gamepedia link point to `Block_of_Coal` for both of those. Then add `CoalOre:Block`, separately.